pipelines/tasks/StarMkref.wdl

```diff
@@ -0,0 +1,31 @@
+
+task StarMkref {
+  File fasta_file  # genome annotation
+  File annotation_file  # gtf annotation file
+
+  # optionally, use a different version of star
+  String star_docker_image = "ambrosejcarr/star-ubam:v1.1.0"
+  
+  command {
+    mkdir genome
+    STAR \
+      --runMode genomeGenerate \
+      --runThreadN $(nproc) \
+      --genomeDir genome \
+      --genomeFastaFiles ${fasta_file} \
+      --sjdbGTFfile ${annotation_file} \
```

Comment:
There are a few of these escaped final newlines in this PR. It's not a big deal, but if somebody tries to add a command after this in the future it's going to break.